Client/qtTeamTalk/utilui.cpp

```diff
@@ -131,6 +131,42 @@ void migrateSettings()
         else if (lang == "Turkish") lc_code = "tr";
         else if (lang == "Vietnamese") lc_code = "vi";
         ttSettings->setValue(SETTINGS_DISPLAY_LANGUAGE, lc_code);
+        // So we should remove old language files
+        QStringList oldFiles = {
+            "/Bulgarian.qm",
+            "/Chinese_Simplified.qm",
+            "/Chinese_Traditional.qm",
+            "/Croatian.qm",
+            "/Czech.qm",
+            "/Danish.qm",
+            "/Dutch.qm",
+            "/English.qm",
+            "/French.qm",
+            "/German.qm",
+            "/Hebrew.qm",
+            "/Hungarian.qm",
+            "/Indonesian.qm",
+            "/Italian.qm",
+            "/Korean.qm",
+            "/Persian.qm",
+            "/Polish.qm",
+            "/Portuguese_BR.qm",
+            "/Portuguese_EU.qm",
+            "/Russian.qm",
+            "/Slovak.qm",
+            "/Slovenian.qm",
+            "/Spanish.qm",
+            "/Thai.qm",
+            "/Turkish.qm",
+            "/Vietnamese.qm"
+        };
+        for (const QString &fileName : oldFiles)
+        {
+            QString filePath = TRANSLATE_FOLDER + fileName;
+            QFile file(filePath);
+            if (file.exists())
+                file.remove();

```

Comment:
You cannot delete these files because they're created by administrator.

Reply:
Yes, but in portable installation or on Linux we can delete files using this method

Reply:
But the portable TeamTalk doesn't have the old language files. It's always a "clean" install.

Reply:
I think that most users copy the files of the new version over the old ones, whether on a portable Windows version or on Linux where the operation is the same. That's why I left this part.
If you think it's unnecessary, we can remove it